widget/embedded/src/components/TypesBlockchain /index.ts

```diff
@@ -0,0 +1 @@
+export { TypesBlockchain } from './TypesBlockchain';
```

Comment:
You put an space at the end of directory name `components/TypesBlockchain /index.ts`. it should be `components/TypesBlockchain/index.ts`